### DISCUSSION TITLE
What is the difference between ActivationAnalyzer and FlopAnaLyzer?

### DISCUSSION THREAD

**Discussion by Anm-pinellia** [open-mmlab/mmengine#1393] [Q&A]
I tried to use Activation Analyzer to analyze my model, but according its name, it seems to count the number of call from acitivation operator. However, it also give the count result about convolution, i can't understand this result.

**Comment by YiyaoYang1** [comment]
ActivationAnalyzer provides access to per-submodule model activation count obtained by tracing a model with pytorch's jit tracing functionality. Activation is defined as the feature quantity produced from one layer. 
FlopAnaLyzer provides access to per-submodule model flop count obtained by tracing a model with pytorch's jit tracing functionality. Flop is usually defined as the number of multiply-accumulate operations.
